src/main/java/org/znerd/confluence/client/http/HttpRequestFactory.java

```diff
@@ -303,6 +303,20 @@ public HttpPost addLabeltoPageRequest(String contentId, String prefix, String la
 
         return postRequest;
     }
+    
+    HttpGet getLabelsByContentIdRequest(String contentId) {
+        assertMandatoryParameter(isNotBlank(contentId), "contentId");
+
+        URIBuilder uriBuilder = createUriBuilder(this.confluenceRestApiEndpoint + "/content/" + contentId + "/label");   
```

Comment:
Why do we need to use a `URIBuilder` if we also use `String` concatenation? 🤔 

It seems to me that if we have a complete `String`, we can use the `URI` class directly, no?